### PR TITLE
clean up all `<p>`, `<dt>` and `<dd>` tag for zh-CN (Part 2)

### PR DESCRIPTION
**Pull request by yin1999**
continue #6088.

### CODE REVIEW

File: files/zh-cn/web/css/background-attachment/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/background-attachment.html")}}</div>
 
-<p class="hidden">该交互式示例网页的源码存储在 GitHub 中，如果你想为该网页项目做点贡献的话，请进此链接 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并给我们一个请求加入的回复。</p>
+<p>该交互式示例网页的源码存储在 GitHub 中，如果你想为该网页项目做点贡献的话，请进此链接 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并给我们一个请求加入的回复。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/background-attachment/index.html:17]: just removing this
```suggestion
```

File: files/zh-cn/web/css/background-clip/index.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/background-clip.html")}}</div>
 
-<p class="hidden">此交互样例的源代码被储存在 GitHub 库中。如果你想要对此交互样例工程有代码贡献，请点击 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a>并向我们发送拉取请求</p>
+<p>此交互样例的源代码被储存在 GitHub 库中。如果你想要对此交互样例工程有代码贡献，请点击 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a>并向我们发送拉取请求</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/background-clip/index.html:18]:
```suggestion
```

File: files/zh-cn/web/css/background-image/index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/background-image.html")}}</div>
 
-<p class="hidden">此交互实例的代码被存储于 Github 仓库。如果你想对这些交互实例贡献代码，请 clone <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并向我们发起拉取请求。</p>
+<p>此交互实例的代码被存储于 Github 仓库。如果你想对这些交互实例贡献代码，请 clone <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并向我们发起拉取请求。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/background-image/index.html:17]:
```suggestion
```

File: files/zh-cn/web/css/block-size/index.html [hunk 4]
```diff
@@ -11,7 +11,7 @@
 
 <div> </div>
 
-<p class="hidden">这个交互例子的源码存在 Githubd 的仓库中。如果你想为这个交互例子的项目做一份贡献，请从 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a>克隆此项目并给我们发送一个 pull 请求。</p>
+<p>这个交互例子的源码存在 Githubd 的仓库中。如果你想为这个交互例子的项目做一份贡献，请从 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a>克隆此项目并给我们发送一个 pull 请求。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/block-size/index.html:14]:
```suggestion
```

File: files/zh-cn/web/css/border-block-end/index.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/border-block-end.html")}}</div>
 
-<p class="hidden">该交互式示例的源码存储在 GitHub 存储库中。如果你想参与该项目的开发，请克隆 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并向我们发送拉取代码请求</p>
+<p>该交互式示例的源码存储在 GitHub 存储库中。如果你想参与该项目的开发，请克隆 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并向我们发送拉取代码请求</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/border-block-end/index.html:12]:
```suggestion
```

File: files/zh-cn/web/css/box-sizing/index.html [hunk 6]
```diff
@@ -38,7 +38,7 @@
 <div class="blockIndicator note">
 <p><strong>Note:</strong> 对于新的 web 站点，你可能希望首先将 box-sizing 设置为 border-box，如下所示：</p>
 
-<p class="syntaxbox">* { box-sizing: border-box; }</p>
+<p>* { box-sizing: border-box; }</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/box-sizing/index.html:41]:
```suggestion
<pre class="brush: css">* { box-sizing: border-box; }</pre>
```

File: files/zh-cn/web/css/filter-function/drop-shadow/index.html [hunk 7]
```diff
@@ -10,7 +10,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/function-drop-shadow.html")}}</div>
 
-<p class="hidden">这个交互式示例的源代码存储在 GitHub 存储库中。如果您想对交互式示例项目作出贡献，请克隆 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并向我们发送拉取请求。</p>
+<p>这个交互式示例的源代码存储在 GitHub 存储库中。如果您想对交互式示例项目作出贡献，请克隆 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并向我们发送拉取请求。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/filter-function/drop-shadow/index.html:13]:
```suggestion
```

File: files/zh-cn/web/css/flex-basis/index.html [hunk 8]
```diff
@@ -16,7 +16,7 @@
 
 
 
-<p class="hidden">这个交互式的例子的源代码存储在一个 GitHub 库。如果你想为这个项目贡献代码，请克隆这个仓库 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 然后给我们提交一个 pull request</p>
+<p>这个交互式的例子的源代码存储在一个 GitHub 库。如果你想为这个项目贡献代码，请克隆这个仓库 <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 然后给我们提交一个 pull request</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/flex-basis/index.html:19]:
```suggestion
```

File: files/zh-cn/web/css/flex/index.html [hunk 9]
```diff
@@ -16,7 +16,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/flex.html")}}</div>
 
-<p class="hidden">这个交互案例的源码存储在 github 仓库。如果你想对此案例做贡献，请克隆项目<a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并且给我们推送一个请求 (PR)。</p>
+<p>这个交互案例的源码存储在 github 仓库。如果你想对此案例做贡献，请克隆项目<a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并且给我们推送一个请求 (PR)。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/flex/index.html:19]:
```suggestion
```

File: files/zh-cn/web/css/font-variant-caps/index.html [hunk 10]
```diff
@@ -14,7 +14,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/font-variant-caps.html")}}</div>
 
-<p class="hidden">该交互式例子的源码位于我们的 Github 仓库中。如果您有兴趣对交互式例子项目进行贡献，请克隆仓库<a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a>，并向我们提出合并请求 (pull request)。</p>
+<p>该交互式例子的源码位于我们的 Github 仓库中。如果您有兴趣对交互式例子项目进行贡献，请克隆仓库<a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a>，并向我们提出合并请求 (pull request)。</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/font-variant-caps/index.html:17]:
```suggestion
```

File: files/zh-cn/web/css/grid-auto-rows/index.html [hunk 11]
```diff
@@ -15,7 +15,7 @@
 
 <div>{{EmbedInteractiveExample("pages/css/grid-auto-rows.html")}}</div>
 
-<p class="hidden">此交互式示例的源代码托管于 GitHub 中。如果您想为交互示例贡献代码，请 clone <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并且给我们发送 pull request.</p>
+<p>此交互式示例的源代码托管于 GitHub 中。如果您想为交互示例贡献代码，请 clone <a href="https://github.com/mdn/interactive-examples">https://github.com/mdn/interactive-examples</a> 并且给我们发送 pull request.</p>
```
Comment (yin1999) [inline files/zh-cn/web/css/grid-auto-rows/index.html:18]:
```suggestion
```

**Review by irvin — CHANGES_REQUESTED** [review]
please rebase

**Pull request merged**